neuro-cli/src/neuro_cli/topics.py

```diff
@@ -458,3 +458,86 @@ async def secrets() -> None:
     `neuro run --volume secret:key:/mount/path/file.txt`.
 
     """
+
+
+@topics.command()
+async def sharing() -> None:
+    """
+    Using sharing functionality
+    ===========================
+
+    Understanding permissions
+    =========================
+    Neu.ro platform supports five levels of access:
+    * deny - no access
+    * list - permits listing entities, but not looking at their details
+    * read - read-only access to entity
+    * write - write-only access (including deletion) to entity
+    * manage - allows modification of entity's permissions
+
+    Please note permissions are inclusive: write permission implies read and manage
+    includes read and write, and so on.
+
+    Permissions can be granted via "`neuro acl grant`" or "`neuro share`" and
```

Comment:
quotas are redundant if you have backticks already.

Reply:
Terminal output without quotes is difficult to read in my opinion. Backticks doesn't change anything visually (at least in my macOS terminal).

Reply:
I believe we should change markdown styling, not the original text.

If you want write the same markdown for web, would you use quotas or backticks only? I think the second.

Reply:
I don't think click's `HelpFormatter` doesn't expose anything that allows formatting of terminal output. I will remove quotes and keep backticks.